pkg/view/component/component.go

```diff
@@ -34,6 +37,15 @@ func NewContentResponse(title []TitleComponent) *ContentResponse {
 	}
 }
 
+// AddTitleComponents Adds any number of components to accompany the title
+func (c *ContentResponse) AddTitleComponents(components ...Component) {
+	for i := range components {
```

Comment:
Why not just assign components to TitleComponents

Reply:
To check first if we are receiving nil components